tests/.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module'
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended'
+  ],
+  env: {
+    browser: true
+  },
+  globals: {},
```

Comment:
I'm not sure why you didn't have to have `percySnapshot` defined here, if eslint still passes then great.

Reply:
I updated the tests to use `import { percySnapshot } from 'ember-percy';`. This imports from the add-on instead of having to use the test helper.

I could either keep this method and remove the test helper, or revert to using the helper and add `percySnapshot` to the eslint globals. I thought it would be better to not use globals. Which do you prefer?

Reply:
Ah very interesting. So our current [docs](https://percy.io/docs/clients/javascript/ember#installation) say to ignore it here and import our `register-helpers` module to install our Ember async helper, so that in acceptance tests you always rely on the global and it works well with the other async helpers like `visit`, `click` etc. This is required in normal acceptance tests so that it doesn't have to be nested in an `andThen()` to get into the right order.

Since you've migrated the tests here to `ember-native-dom-helpers` and are using `await` it may not matter anymore in this case, the only slightly weird part I can think of is that we like using this repository as an example of writing Percy tests the recommended way. I'm not sure of the state of grand testing unification, but we'll need to support the Ember async helpers workflow for a long while.

Just some thoughts, I don't really have strong feelings here either way — just wanted to point out the effect on our docs and our currently recommended workflow for acceptance tests. What do you think we should do here?

Reply:
Thanks for the detailed explanation. I agree that consistency and being able to use this repo as an example of writing Percy tests is important. So with that in mind I added the `percySnapshot` global back in for tests. I think it makes sense to keep and support the async helper.